app/controllers/project.controller.js

```diff
@@ -37,37 +35,42 @@ const post = async (req, res) => {
 
 const put = async (req, res) => {
   try {
-    if (!isAuthorizedUser(req)) {
+    const project = await getProject(req, res);
+    if (project.loginId !== req.user.id) {
```

Comment:
This is always going to be true since project.loginId is what's passed in as req.user.id.
Suggest referencing "const post" comparison logic above.
Suggest moving getProject call after the comparison to potentially save cycles.